packages/terra-button-group/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+
+*Fixed
+  *Fixed Terra- Button Group Accessiblity Issues.
```

Comment:
This is too vague. What ate the accessibility issues you fixed?

Reply:
Updated doc file